Src/Database/Services/secretKey.js

```diff
@@ -0,0 +1,3 @@
+export const secret = {
+  env: "richardsecret",
```

Comment:
This is not a service!!!
You must also move all your secrets into `.env`...
DON'T PUSH YOUR SECRETS TO GITHUB!!!